package/network/utils/ltq-dsl-base-files/Makefile

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (C) 2019 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ltq-dsl-base-files
```

Comment:
I'd prefer just "ltq-dsl-base"

Reply:
I have chosen the name in reference to the origin of the files it contains, namely /target/linux/lantiq/base-files.

Reply:
It's nitpicking, but essentially this is about what this package is supposed to evolve into:

ltq-dsl-base: This means it's the "base package" for the other two packages, and may contain whatever these packages would reasonably share (e.g. one could also move dependencies, setting kernel symbols etc.)

ltq-dsl-base-files: This means the package is limited to containing "files", i.e. shared scripts and library files.

Of course, this is nitpicking, and most probably nobody except me will really care about that :-)

Reply:
OK, I agree.

Reply:
Now, after I've finally offered you the option to choose :-)

Reply:
I agree with you because I like the "not just files" argument.